python_modules/airline-demo/airline_demo/resources.py

```diff
@@ -142,3 +152,115 @@ def _do_load(data_frame, table_name):
         dialect='postgres',
         load_table=_do_load,
     )
+
+
+class AbstractFilesystemManager(six.with_metaclass(ABCMeta)):  # pylint: disable=no-init
```

Comment:
Cool. We will have to consolidate this with the run storage part, but I think that will be straightforward.

Reply:
thrilled to rip this stuff out in favor of a common storage/filesystem system